src/open_inwoner/components/templates/components/Header/NavigationAuthenticated.html

```diff
@@ -7,7 +7,7 @@
     <nav class="primary-navigation primary-navigation--desktop primary-navigation__authenticated" aria-label="Navigatie na inloggen">
         <ul class="primary-navigation__list">
             <li class="primary-navigation__list-item">
-            {% button text=_('Welkom ')|addstr:request.user.display_name type="button" icon="expand_more" icon_position="after" icon_outlined=True transparent=True extra_classes="primary-navigation--toggle" %}
+            {% button text=_('Welkom ')|addstr:request.user.get_short_name type="button" icon="expand_more" icon_position="after" icon_outlined=True transparent=True extra_classes="primary-navigation--toggle" %}
```

Comment:
See previous comment.

Additionally, if you follow the suggestion about `display_name`, you can delete `get_short_name` from the User model